components/Datepicker/src/index.tsx

```diff
@@ -0,0 +1,400 @@
+            { locale: locale },
+          );
+          if (!isSameMonth(newDate, subMonths(state.current, 1))) {
+            newDate = subWeeks(newDate, 1);
+          }
+          setState({ ...state, current: newDate });
+        }
+        break;
+      case 'PageDown':
+        e.preventDefault();
+        if (e.shiftKey) {
+          let newDate = setDay(
+            addWeeks(setDate(addYears(state.current, 1), 1), getWeekOfMonth(state.current, { locale: locale }) - 1),
+            getDay(state.current),
+            { locale: locale },
+          );
+          if (!isSameMonth(newDate, addYears(state.current, 1))) {
+            newDate = subWeeks(newDate, 1);
+          }
+          setState({ ...state, current: newDate });
+        } else {
+          let newDate = setDay(
+            addWeeks(setDate(addMonths(state.current, 1), 1), getWeekOfMonth(state.current, { locale: locale }) - 1),
+            getDay(state.current),
+            { locale: locale },
+          );
+          if (!isSameMonth(newDate, addMonths(state.current, 1))) {
+            newDate = subWeeks(newDate, 1);
+          }
+          setState({ ...state, current: newDate });
+        }
+        break;
+      case 'Escape':
+        setState({ ...state, opened: false });
+        break;
+      case 'Tab':
+        if (!e.shiftKey) {
+          e.preventDefault();
+          backButtonRef.current?.focus();
+          break;
+        }
+    }
+  };
+
+  const renderDay = (i: number, j: number) => {
+    const date = setDay(addWeeks(setDate(state.current, 1), i), (j + (locale.options?.weekStartsOn || 0)) % 7, {
+      locale: locale,
+    });
+    if (isSameMonth(date, state.current)) {
+      return (
+        // eslint-disable-next-line jsx-a11y/role-supports-aria-props
+        <button
+          type="button"
+          className={clsx('denhaag-datepicker__calendar-day', {
+            'denhaag-datepicker__calendar-day--current': isEqual(today, date),
+            'denhaag-datepicker__calendar-day--selected': state.selected && isEqual(state.selected, date),
+          })}
+          tabIndex={isEqual(state.current, date) ? 0 : -1}
+          ref={isEqual(state.current, date) ? currentButtonRef : null}
+          onClick={() => {
+            setState({ ...state, selected: date, opened: false });
+          }}
+          onKeyDown={onKeyDownDay}
+          aria-selected={state.selected && isEqual(state.selected, date) ? 'true' : undefined}
+        >
+          {date.getDate()}
+        </button>
+      );
+    }
+    return '';
+  };
+
+  return (
+    <div className={rootStyles} ref={ref}>
+      <button
+        type="button"
+        className="denhaag-datepicker__button"
+        aria-label={openCalendarLabel}
+        onClick={() => {
+          setState({ ...state, opened: !state.opened });
+        }}
+        tabIndex={props.disabled ? -1 : 0}
+        ref={openButtonRef}
+      >
+        <CalendarIcon />
+      </button>
+      <input
```

Comment:
On safari, the date separators in the placeholder are darker, while on chrome they're the entire placeholer is grey. 
**Safari:** 
![image](https://user-images.githubusercontent.com/24610692/147241958-54121920-7e6c-4549-923e-2acc60287674.png)
**Chrome**
![image](https://user-images.githubusercontent.com/24610692/147242012-e7c7fc5d-b3d2-4f6b-a50f-c71a5f2fac3c.png)

Reply:
Reworked the placeholders to work in all browsers, so this does not occur anymore